src/cdi/CPEloss.hh

```diff
@@ -129,7 +132,40 @@ public:
 	 * \brief Returns the general Eloss model type (Analytic),
 	 * defined in the enum at the top of this file.
 	 */
-  virtual rtt_cdi::CPModelType getModelType() const = 0;
+  //virtual rtt_cdi::CPModelType getModelType() const = 0;
+
+  /*!
+   * \brief Query to determine the transporting particle type.
+   */
+  CParticle getProjectile() const { return projectile; }
+
+  /*!
+   * \brief Query to determine the target species type.
+   */
+  CParticle getTarget() const { return target; }
+
+  /*!
+   * \brief Query to determine the CP Model type.
+   */
+  CPModelType getModelType() const { return model_type; }
+
+  /*!
+   * \brief Query to determine the angle cutoff used by the CP model.
+   */
+  CPModelAngleCutoff getModelAngleCutoff() const { return model_angle_cutoff; }
+
+protected:
+  // Particle being transported i.e. subject to energy loss
+  CParticle projectile;
+
+  // Target particle
+  CParticle target;
+
+  // CP Model type
+  CPModelType model_type;
+
+  // CP Model angle cutoff
+  CPModelAngleCutoff model_angle_cutoff;
 };
 
 } // namespace rtt_cdi
```

Comment:
Our style guide requires a 3 line footer on headers and implementation files.

Reply:
Fixed in `cdi_analytic/Analytic_CP_Eloss.hh`, `cdi_analytic/Analytic_CP_Eloss.cc`, `cdi_CPEloss/Tabular_CP_Eloss.hh', and `cdi_CPEloss/Tabular_CP_Eloss.cc'.